ui/.prettierrc

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
@andrashee Yeah, I was confused it was not already in the root folder but thought maybe there is a reason for it, so that's why I didn't move it. Maybe whoever works on the native app would have to tell if this could have a negative effect on their editor if we move it to the root.